src/components/HTMLEngineProvider/HTMLRenderers/PreRenderer.js

```diff
@@ -54,19 +48,19 @@ const BasePreRenderer = forwardRef((props, ref) => {
                         role={CONST.ROLE.PRESENTATION}
                         accessibilityLabel={props.translate('accessibilityHints.prestyledText')}
                     >
-                        <View>
-                            {/* eslint-disable-next-line react/jsx-props-no-spreading */}
-                            <TDefaultRenderer {...defaultRendererProps} />
-                        </View>
+                        {/* <View> */}
```

Comment:
Some left-over, I assume?

Reply:
Also, is this change related to our issue?

While I agree that the wrapping `View` _looks_ unnecessary, it would require investigation to ensure this is unneeded. If removing this causes a regression, we'll have to take responsibility.